packages/db/src/loaders/schema/index.ts

```diff
@@ -0,0 +1,55 @@
+import { TruffleDB } from "@truffle/db";
+import { ArtifactsLoader } from "./artifactsLoader";
+export { ArtifactsLoader };
+import { schema as rootSchema } from "@truffle/db/schema";
+import { Workspace, schema } from "@truffle/db/workspace";
+const tmp = require("tmp");
+import { makeExecutableSchema } from "@gnd/graphql-tools";
+import { gql } from "apollo-server";
+
+//dummy query here because of known issue with Apollo mutation-only schemas
```

Comment:
Not a blocker but a link to the issue would be helpful for our future selves.

Reply:
my present self doesn't remember what this is about. was this even me? @fainashalts was this you?

Reply:
I remove the comment in the meantime. If we come up with a source, we can add it back in... might be helpful!